From 53cfe9a59aaec8e0fd128635fbaebc0c610b73da Mon Sep 17 00:00:00 2001
From: DamsRoxxx <walzdamien@hotmail.com>
Date: Sat, 20 Oct 2018 10:39:21 +0200
Subject: [PATCH] Fix : Conflict with Game Focus function

The ScrollLock key was mapped to inhibit screen function of puae.

Removed the mapping to allow Game Focus function of retroarch to work.
---
 libretro/libretro-keymap.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libretro/libretro-keymap.h b/libretro/libretro-keymap.h
index 504c25c..8264f36 100644
--- a/libretro/libretro-keymap.h
+++ b/libretro/libretro-keymap.h
@@ -142,7 +142,7 @@ const int keyboard_translation[512] =
 	/* Key state modifier keys */
 	-1,      /* SDLK_NUMLOCK = 300 */
 	AK_CAPSLOCK,    /* SDLK_CAPSLOCK = 301 */
-	AKS_INHIBITSCREEN,    /* SDLK_SCROLLOCK = 302 */
+	-1,    /* SDLK_SCROLLOCK = 302 */
 	AK_RSH,    /* SDLK_RSHIFT = 303 */
 	AK_LSH,    /* SDLK_LSHIFT = 304 */
 	AK_RCTRL,    /* SDLK_RCTRL = 305 */
